AirCasting/SessionsSynchronization/Controller/Interfaces/SessionDownstream.swift

```diff
@@ -0,0 +1,55 @@
+// Created by Lunar on 16/06/2021.
+//
+
+import Foundation
+import Combine
+
+/// Defines interface for objects which provide downstream connection for sessions
+///
+/// Overview of the sync process:
+/// 1. Fetch sessions stored on a device
+/// 2. Diff them against upstream database
+/// 3. Download/Upload/Delete accordingly (this interface)
+protocol SessionDownstream {
+    func download(session: SessionUUID) -> AnyPublisher<SessionsSynchronization.SessionDownstreamData, Error>
+}
+
+// MARK: Data structures
```

Comment:
Can't find it without it ;)

Reply:
![tenor](https://user-images.githubusercontent.com/10663361/123439263-da2a0480-d5d1-11eb-87ce-01d9a7a669cf.gif)